src/redux/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+export const categorySlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    checkStatus: (state) => [...state, { msg: 'Under Construction' }],
+  },
+});
+
+export const { checkStatus } = categorySlice.actions;
+
+export default categorySlice.reducer;
```

Comment:
- [ ] Good job so far but from the project requirement [ducks pattern](https://github.com/erikras/ducks-modular-redux)...` const CREATE = 'my-app/widgets/CREATE';` so you should have something like this `const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS'; `. Please fix it :+1: 

#### Find an example below :arrow_down: 

```

// Actions
const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';

// Reducer
export default function categoriesReducer(state = [], action) {
  switch (action.type) {
    case CHECK_STATUS:
      return 'Under Construction';
    default:
      return state;
  }
}

// Action creators
export function checkStatus() {
  return {
    type: CHECK_STATUS,
  };
}

```

Reply:
Hi @codecaiine , 
Thank you for your review but I have a few concerns.
I thought the dunk patter is just a way of arranging directories and files. Because i noticed it was stated in the requirements to use redux toolkit which made me write my code in this manner. Please can you explain further?